src/database/migrations/20200906145335-create-state.js

```diff
@@ -0,0 +1,40 @@
+export default {
+  up(queryInterface, Sequelize) {
+    return queryInterface.sequelize
+      .query('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";')
+      .then(() =>
+        queryInterface.createTable("States", {
+          countryId: {
+            type: Sequelize.STRING,
+            allowNull: false,
+          },
+          id: {
+            allowNull: false,
+            primaryKey: true,
+            type: Sequelize.UUID,
+            defaultValue: Sequelize.literal("uuid_generate_v4()"),
+          },
+          name: {
+            type: Sequelize.TEXT,
```

Comment:
The same here too, the capital and gallery...I think the types should be .STRING